fixed user account drop down, removed unusec wallet component, move u…

KatarzynaStefanowiczPagepro opened the pull request:
…ser info avatar to separate component

## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/JoinColony/colonyCDapp?pullRequest=764) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/JoinColony/colonyCDapp?pullRequest=764) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/JoinColony/colonyCDapp?pullRequest=764) it.</sub>

KatarzynaStefanowiczPagepro commented:
https://tw.auroracreation.com/app/tasks/15516473

Fixed the user account drop-down,
Removed unused wallet component
Moved the user info avatar to separate, more reusable component 

![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/f0243ec8-709c-4bf6-a304-e8a57b7b1600)

krawczykkatarzyna commented on src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx:
```diff
@@ -109,61 +108,51 @@ const UserMenu: FC<UserMenuProps> = ({
+                ) : (
+                  <button
+                    type="button"
+                    className="navigation-link"
```
aria-label is missing

KatarzynaStefanowiczPagepro replied:
aria-label isn't required when inside the button is text, here is a item name

krawczykkatarzyna commented on src/components/common/Extensions/UserNavigation/partials/UserSubmenu/UserSubmenu.tsx:
```diff
@@ -12,8 +12,14 @@ const UserSubmenu: FC<UserSubmenuProps> = ({ submenuId }) => (
+              appearance={{ size: 'small' }}
+              className="navigation-link"
+            />
+            <p className="ml-2 ">{label}</p>{' '}
```
{' '} is it needed?

KatarzynaStefanowiczPagepro replied:
Missclick :)
